spec.html

```diff
@@ -3147,7 +3147,7 @@ <h2>[[GetOwnProperty]] ( _P_ )</h2>
             The normal return type is either Property Descriptor or Undefined.
           </li>
           <li>
-            If the Type of the return value is Property Descriptor, the return value must be a fully populated Property Descriptor.
+            If the return value is a Property Descriptor, the return value must be a fully populated Property Descriptor.
```

Comment:
These changes could've been a separate PR or could've been saved for the PR that removes the Type AO completely. They're a bit out of place in this PR.

Reply:
Sure, I can extract them to another PR, nbd

Reply:
→ #3410